src/App.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <HomePage />
+</template>
+
+<script>
+import HomePage from "./pages/Homepage.vue";
```

Comment:
Since this component is simply displaying the `HomePage`, can't it be deleted and you can directly use the `HomePage`?

Reply:
Can do.

Reply:
Agree. Because this is a very simple webapp for now, we can keep the home ka content here only